CONTRIBUTING.md

```diff
@@ -101,3 +101,75 @@ This simply ensures that you have the legal right to make the contribution.
 
 Please feel free to add yourself to the CONTRIBUTORS.md file with your pull request.
 We appreciate your efforts and would love to add you to our list of contributors.
+
+## Tooling
+
+We use linters to standardize and maintain style in our files. 
+These linters are added as GitHub Actions to verify that each pull request is compatible.
+In this section, we provide instructions to detail how you can run these linters locally.
+
+In addition, we also describe the steps to deconstruct the rules to help their adoption for other use cases.
+
+### yamllint
```

Comment:
I forgot typos

https://github.com/crate-ci/typos

And the related VScode extension

https://marketplace.visualstudio.com/items?itemName=tekumara.typos-vscode

Reply:
I added that one.